### ISSUE TITLE
How to fine-tune SAM 3 with SAM3InteractiveImagePredictor (for SAM 1 task)?

### ISSUE THREAD

**Issue by MaxTeselkin** [facebookresearch/sam3#386]
Hi!

I am curious how can I fine-tune SAM 3 with SAM3InteractiveImagePredictor to use fine-tuned checkpoints in interactive mode (positive and negative clicks)?

Currently fine-tuned checkpoints do not have keys in state dict required by SAM3InteractiveImagePredictor and can be loaded only with enable_inst_interactivity = False, even if model was fine-tuned with enable_segmentation = True (with semantic segmentation loss).

I understand that it is necessary to enable instance interactivity in training config, so model configuration should look like this:

```
model:
    _target_: sam3.model_builder.build_sam3_image_model
    bpe_path: ${paths.bpe_path}
    device: cpus
    eval_mode: false
    enable_segmentation: ${scratch.enable_segmentation} # Warning: Enable this if using segmentation.
    enable_inst_interactivity: True
```

It gives additional 20 M of params, so model summary will print 860 M instead of 840. But these additional parameters require specific loss function to fine-tune - based on sam 2 repo, I guess [this](https://github.com/facebookresearch/sam3/blob/main/sam3/train/loss/loss_fns.py#L777) loss function must be used - MultiStepMultiMasksAndIous.

The problem here is the fact that this loss function [requires](https://github.com/facebookresearch/sam3/blob/main/sam3/train/loss/loss_fns.py#L824) outputs dictionary to have keys like "multistep_pred_multimasks_high_res", "multistep_pred_ious" and "multistep_object_score_logits", which are not present in outputs dictionary which model returns with example training configs you provided.

So how to make model predict such output dictionaries?
